### ISSUE TITLE
The rewards panel design is not responsive.

### ISSUE THREAD

**Issue by gdregalo** [brave/browser-android-tabs#890]
The design of html content of the rewards panel is not responsive to different viewport sizes. On portrait orientation it leaves some unused strip on the right. And while in landscape orientation, it occupies more room then available.

**Comment by ryanml** [comment]
@gdregalo can you post screenshots?

@serg this sounds like something covered in the scope of https://github.com/brave/browser-android-tabs/issues/865 correct?

**Comment by ryanml** [comment]
The rewards panel is not meant to support different viewport sizes, really is only targeted for 1 width. The height of it may change depending on whether the summary is showing

**Comment by gdregalo** [comment]
The background of `<html>` element was made `red` for better visibility.
![screenshot_20181126-111104](https://user-images.githubusercontent.com/37560650/49029725-4ec70900-f173-11e8-98cb-38b796c817b3.png)
![screenshot_20181126-111053](https://user-images.githubusercontent.com/37560650/49029731-54245380-f173-11e8-8876-364925d92ae2.png)

**Comment by ryanml** [comment]
@gdregalo is this then a duplicate issue of https://github.com/brave/browser-android-tabs/issues/832?

**Comment by petemill** [comment]
@gdregalo what is the logic defining the width of the panel container? Is it fixed dp, or is it a percentage of the available space? I heard there was a plan to have it be dynamic according to what the size the html defines like desktop Chrome, but I could be wrong? In the code right now, on 1 device it is always 376x696 no matter what the html states.
As far as I know @jenn-rhim and @ryanml decided that the panel content will *not* be responsive. Therefore it will be a fixed-width in dp and a max-height in dp (the html will scroll when that height is not met). I'm not sure _why_ we're not making it responsive but that's the design call as far as I know 🤷‍♂️.

**Comment by ryanml** [comment]
Yes it was my understanding through our design working session that the panel content is targeted for a single size, @jenn-rhim feel free to chime in if anyone is incorrect.

**Comment by petemill** [comment]
Single width and height for **both** welcome and popup screens, and all languages @jenn-rhim?

**Comment by ryanml** [comment]
This is now part of the native panel Ui
